Test/PcapDeviceTest.cs

```diff
@@ -167,6 +169,21 @@ public void Immediate([Values] bool? immediate)
             }));
         }
 
+        /// <summary>
+        /// Test that we can retrieve packet information via GetNextPacketPointers
+        /// </summary>
+        [Test]
+        public void GetNextPacketPointers()
+        {
+            using var device = TestHelper.GetPcapDevice();
+            device.Open();
+
+            // confirm that we can retrieve the next packet pointers
+            var header = IntPtr.Zero;
+            var data = IntPtr.Zero;
+            device.GetNextPacketPointers(ref header, ref data);
```

Comment:
Assert the header is not `IntPtr.Zero` ?

Reply:
good call, let me do that